lib/auth/auth.go

```diff
@@ -4867,7 +4867,7 @@ func (a *Server) submitAccessReview(
 
 	// final permission checks and review application must be done by the local backend
 	// service, as their validity depends upon optimistic locking.
-	req, err := a.ApplyAccessReview(ctx, params, checker)
+	req, err := a.ApplyAccessReview(ctx, params, checker, a.clock)
```

Comment:
`ApplyAccessReview` is a method on `a` so it already has access to `a.clock`. I would avoid passing it in here as it makes the API more confusing (what happens if we pass in some other clock, for example?)

Reply:
thanks for the catch! my original attempt at this required the clock but not anymore